redis/content.md

```diff
@@ -26,10 +32,24 @@ For more about Redis Persistence, see [http://redis.io/topics/persistence](http:
 
 ## connecting via `redis-cli`
 
+Create the network first:
+
 ```console
-$ docker run -it --network some-network --rm %%IMAGE%% redis-cli -h some-redis
+$ docker network create some-network
+```
```

Comment:
Along similar lines, the `docker network create` bits were intentionally left out of https://github.com/docker-library/docs/pull/1441 (as implied by the title) -- we'd prefer not to document Docker basics here and instead provide more reference/tips for using Redis with this image.